src/test/groovy/com/skalashynski/spring/springboot/util/ResourceUtils.java

```diff
@@ -1,59 +0,0 @@
-package com.skalashynski.spring.springboot.util;
-
-import java.io.File;
-import java.io.IOException;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Path;
-
-import static java.util.Objects.requireNonNull;
-
-/**
- * Utility class containing common methods around bundled resources in support of testing.
- */
-public class ResourceUtils {
-
-    /**
```

Comment:
I'd not recommend you remove this class. This is really necessary and useful class. Keep this as it is

Reply:
Why should we use this class and instead of "new File(...).text" as we did in the repository tests?